src/popup/router/components/InputField.vue

```diff
@@ -142,6 +145,13 @@ export default {
       return !!this.message;
     },
   },
+  methods: {
+    checkIfNumber(event) {
+      if (!(this.type !== 'number' || ['Enter', 'Backspace'].includes(event.key)) && !/^[0-9/-]$/.test(event.key)) {
```

Comment:
I suggest to avoid double negations, especially as the code should target the type number fields.
Maybe this will be OK:
```
if (this.type === 'number' && !['Enter', 'Backspace'].includes(event.key) && !/^[0-9/-]$/.test(event.key)) {
```
(not tested, only suggestion)